src/jdk.internal.vm.ci/share/classes/jdk/vm/ci/meta/ConstantPool.java

```diff
@@ -191,6 +191,23 @@ interface BootstrapMethodInvocation {
          * @jvms 5.4.3.6
          */
         List<JavaConstant> getStaticArguments();
+
+        /**
+         * If this bootstrap method invocation is for a {@code
+         * CONSTANTAdd_InvokeDynamic_info} pool entry, then this method ensures the
+         * invoke dynamic is resolved. This can be used to compile time resolve the
```

Comment:
What exactly does resolving an invoke dynamic mean?
Also I would leave out the sentence about "compile time" unless you clarify exactly what that means.